javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/NoopLoggingCustomizer.java

```diff
@@ -5,8 +5,15 @@
 
 package io.opentelemetry.javaagent.tooling;
 
-enum NoopLoggingCustomizer implements LoggingCustomizer {
-  INSTANCE;
+import com.google.auto.service.AutoService;
+
+@AutoService(LoggingCustomizer.class)
+public final class NoopLoggingCustomizer implements LoggingCustomizer {
+
+  @Override
+  public String name() {
+    return "none";
```

Comment:
This constant might be checked elsewhere, but  it might be slightly less fragile if we call it `noop` instead of `none`.